pandas_td/test_td.py

```diff
@@ -102,30 +98,33 @@ def tearDown(self):
         os.environ.clear()
         os.environ.update(self._environ)
 
-    @raises(ValueError)
     def test_error_without_parameters(self):
-        Connection()
+        try:
```

Comment:
Personally, it's explicit to use `assertRaises` like:

```py
with self.assertRaises(ValueError):
    Connection()
```

Reply:
Because it's written in unittest.